dist/index.js

```diff
@@ -41248,48 +41248,29 @@ const run = async () => {
   core.startGroup('Extracting action context');
 
   // Setup Runner Context
-  const envRunnerFilters = [
-    // Don't strip RUNNER_ and GITHUB_ env vars so we can avoid naming conflicts
-    { prefix: 'RUNNER_', strip: false },
-  ];
+  // Don't strip RUNNER_ and GITHUB_ env vars so we can avoid naming conflicts
   const runnerKey = 'RUNNER_TRACKING_ID';
-
   const githubRunnerCtx = process.env[runnerKey]
     ? {
-        GithubRunner: {
-          key: process.env[runnerKey],
-          ...createContext(envRunnerFilters, runnerKey),
-        },
+        GithubRunner: createContext(process.env[runnerKey], { prefix: 'RUNNER_', strip: false }, runnerKey),
       }
     : {};
 
   // Setup Github Context
-  const envGithubFilters = [{ prefix: 'GITHUB_', strip: false }];
   const githubKey = 'GITHUB_REPOSITORY';
   const githubCtx = process.env[githubKey]
     ? {
-        Github: {
-          key: process.env[githubKey] ? process.env[githubKey].split('/').pop().trim() : process.env['GITHUB_RUN_ID'],
-          ...createContext(envGithubFilters, githubKey),
-        },
+        Github: createContext(
+          process.env[githubKey].split('/').pop().trim(),
+          { prefix: 'GITHUB_', strip: false },
+          githubKey,
+        ),
       }
     : {};
 
   // Setup LaunchDarkly Context
-  const envLDFilters = [{ prefix: 'LD_', strip: true }];
```

Comment:
Originally we were doing this:

```
      GithubCustomAttributes: {
        key: contextKey ? contextKey : Date.now(),
        ...createContext(envLDFilters),
      },
```

to build the context. This allows a customer to theoretically overwrite the key we are trying to explicitly set (and potentially to something invalid) by setting a variable to `LD_key=my invalid key`.

This also means that you could have overridden this with a valid key, and generated a context with only has the key set.

So for my change, I modified it so the key we provide is **always** the one used, and I also changed it so that we always generated a gh custom attributes context. This way we don't lose the ability to have a context that is ONLY a key, and people who don't want to use that context can simply not target that kind.